scripts/PearlPowderISIS/pearl_cycle_factory.py

```diff
@@ -0,0 +1,154 @@
+# pylint: disable=anomalous-backslash-in-string, too-many-branches, superfluous-parens
+
+from mantid.simpleapi import *
+import pearl_routines
+
+
+def get_cycle_dir(number, currentdatadir):
+    if type(number) is int:
+        runno = number
+    else:
+        num = number.split("_")
+        runno = int(num[0])
+
+    print "The run number is get cycle is", runno
+    if (runno < 71009):
+        cycle = "10_2"
+        print "cycle is set to", cycle
+        instver = "old"
+        datadir = "X:\data\cycle_" + cycle + "\\"
```

Comment:
@FedeMPouzols I should simplify that but to set the raw directory you can use the following command.. 
```
currentDir = pearl_routines.pearl_set_currentdatadir("\\olympic\\Babylon5\\Public\\Shahroz\\PEARL\\RawFiles\\")
pearl_routines.PEARL_setdatadir(currentDir)
```
They seem to have two global variables doing the same thing `pearl_datadir` and `currentdatadir`, hence you would have to set it twice, I'll take a look at this in one of the follow up issues when system test is in place.